### ISSUE TITLE
[BUG] <title> Circular Progress Indicator overlayed on top of videos

### ISSUE THREAD

**Issue by MjnMixael** [sarbagyastha/youtube_player_flutter#971]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package

youtube_player_flutter

### What happened?

The latest version of this package seems to overlay a circular progress indicator on top of all videos in my app. [See the video here.](https://youtube.com/shorts/GP_V205nQ88?feature=share)

### What is the expected behaviour?

No circular progress indicator

### How to reproduce?

My code is show a video is very simple.

`import 'package:flutter/material.dart';
import 'package:youtube_player_flutter/youtube_player_flutter.dart';

class VideoPlayerOverlay extends StatelessWidget {
  final String videoUrl;
  final Function() onClose;

  const VideoPlayerOverlay({
    super.key,
    required this.videoUrl,
    required this.onClose,
  });

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      backgroundColor: Colors.transparent, // Make background transparent
      body: Stack(
        children: [
          // Video player positioned on top
          Positioned.fill(
            child: YoutubePlayer(
              controller: YoutubePlayerController(
                initialVideoId: videoUrl,
                flags: const YoutubePlayerFlags(
                  autoPlay: true,
                  mute: false,
                ),
              ),
              bottomActions: [
                IconButton(
                  icon: const Icon(Icons.close),
                  onPressed: () => Navigator.pop(context),
                ),
                const CurrentPosition(),
                const ProgressBar(isExpanded: true),
              ],
              showVideoProgressIndicator: true,
              onEnded: (result) => Navigator.pop(context), // Close overlay on video end
            ),
          ),
        ],
      ),
    );
  }
}`

### Flutter Doctor Output

```shell
[√] Flutter (Channel stable, 3.22.2, on Microsoft Windows [Version 10.0.19045.4651], locale en-US)
    • Flutter version 3.22.2 on channel stable at C:\Users\mjnk\dev\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 761747bfc5 (10 weeks ago), 2024-06-05 22:15:13 +0200
    • Engine revision edd8546116
    • Dart version 3.4.3
    • DevTools version 2.34.3

[!] Windows Version (Problem detected with Windows installation)
    ! Get-Process failed to complete

[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Users\mjnk\AppData\Local\Android\sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.9+0--11185874)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[!] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.9.6)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.9.34728.123
    ! Unable to locate a Windows 10 SDK. If building fails, install the Windows 10 SDK in Visual Studio.

[√] Android Studio (version 2023.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.9+0--11185874)

[√] VS Code (version 1.89.1)
    • VS Code at C:\Users\mjnk\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.88.0

[√] Connected device (3 available)
    • Pixel 8 Pro (mobile) • adb-38310DLJG0052Y-v2ZxeJ._adb-tls-connect._tcp • android-arm64  • Android 14 (API 34)
    • Windows (desktop)    • windows                                         • windows-x64    • Microsoft Windows
      [Version 10.0.19045.4651]
    • Chrome (web)         • chrome                                          • web-javascript • unknown

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 2 categories.
```

**Comment by AliEsmaeil** [comment]
Yes, recently I have encountered this issue, it's considered as a bug for the latest versions. I have tried to use older versions like `8.1.2` according to someone suggestion and it's working. for more info you can find my [question](https://stackoverflow.com/questions/78859241/flutter-youtube-video-keeps-loading/78860457) on SO.

**Comment by sarbagyastha** [comment]
Fixed with v9.0.3
